openquake/hazardlib/calc/disagg.py

```diff
@@ -114,15 +114,24 @@ def _eps3(truncation_level, n_epsilons):
 
 
 # this is inside an inner loop
-def disaggregate(ctxs, tom, g_by_z, iml2dict, eps3, sid=0, bin_edges=()):
+def disaggregate(ctxs, tom, g_by_z, iml2dict, eps3, sid=0, bin_edges=(),
+                 epsstar=False):
     """
     :param ctxs: a list of U RuptureContexts
     :param tom: a temporal occurrence model
     :param g_by_z: an array of gsim indices
     :param iml2dict: a dictionary of arrays imt -> (P, Z)
     :param eps3: a triplet (truncnorm, epsilons, eps_bands)
+    :param sid:
+    :param bin_edges:
+    :param epsstar: a boolean. When True, disaggregation contains eps* results
     """
     # disaggregate (separate) PoE in different contributions
+    # U - Number of contexts (i.e. ruptures)
+    # E - Number of epsilon bins between lower and upper truncation
+    # M - Number of IML values to disaggregate
+    # P -
```

Comment:
P is the number of PoEs in poes_disagg
Z the number of realizations to consider